test/JDBC/expected/BABEL-3407-vu-prepare.out

```diff
@@ -0,0 +1,84 @@
+CREATE TABLE babel_3407_table1 (
+[id] INT,
+[name] VARCHAR(25),
+[state] VARCHAR(25) );
+GO
+
+INSERT INTO babel_3407_table1 values
+(1,'Divya',NULL),
+(2,'Akshay','Bengaluru'),
+(3,'Kavya','Kolkata');
+GO
+~~ROW COUNT: 3~~
+
+
+-- FOR JSON PATH clause
+CREATE VIEW babel_3407_view1 AS
+SELECT id, name, state
+FROM babel_3407_table1
+FOR JSON PATH;
+GO
+
+-- ROOT directive without specifying value
+CREATE VIEW babel_3407_view2 AS
+SELECT name, state
+FROM babel_3407_table1
+FOR JSON PATH, ROOT;
+GO
+
+-- ROOT directive with specifying ROOT value
+CREATE VIEW babel_3407_view3 AS
+SELECT name, state
+FROM babel_3407_table1
+FOR JSON PATH, ROOT('Employee');
+GO
+
+-- WITHOUT_ARRAY_WRAPPERS directive
+CREATE VIEW babel_3407_view4 AS
+SELECT name, state
+FROM babel_3407_table1
+FOR JSON PATH, WITHOUT_ARRAY_WRAPPER;
+GO
+
+-- INCLUDE_NULL_VALUES directive
+CREATE VIEW babel_3407_view5 AS
+SELECT name, state
+FROM babel_3407_table1
+FOR JSON PATH, INCLUDE_NULL_VALUES;
+GO
+
+-- Multiple Directives
+CREATE VIEW babel_3407_view6 AS
+SELECT id, name, state
+FROM babel_3407_table1
+FOR JSON PATH, ROOT('Employee'), INCLUDE_NULL_VALUES;
+GO
+
+CREATE VIEW babel_3407_view7 AS
+SELECT id, name, state
+FROM babel_3407_table1
+FOR JSON PATH, WITHOUT_ARRAY_WRAPPER, INCLUDE_NULL_VALUES;
+GO
+
+-- Throws error as ROOT and WITHOUT_ARRAY_WRAPPER cannot be used together
+CREATE VIEW babel_3407_view8 AS
```

Comment:
`CREATE VIEW` should fail in this case.

Reply:
It is throwing an error. It can be seen in BABEL-3407-vu-verify.out.

Reply:
It should throw an error for `CREATE VIEW`, not for the select statement. Please check T-SQL behavior.

Reply:
Resolved in latest commit.